lib/srv/reexec_linux.go

```diff
@@ -82,7 +84,27 @@ func userCommandOSTweaks(cmd *exec.Cmd) {
 	}
 	// Linux only: when parent process (this process) dies unexpectedly, kill
 	// the child process instead of orphaning it.
-	// SIGKILL because we don't control the child process and it could choose
+	// SIGKILL because we don't control the child process, and it could choose
 	// to ignore other signals.
 	cmd.SysProcAttr.Pdeathsig = syscall.SIGKILL
 }
+
+// adjustOOMScore sets the OOM score for the current process to 0. This value
```

Comment:
It is not really clear what it means to set the score to 0. Perhaps instead of `adjustOOMScore`, it could be called `resetOOMScore` or `setNeutralOOMScore` which makes it clearer that you are setting the score back to the default?

Reply:
Good point. I'll rename it.